content/courses/native-onchain-development/paging-ordering-filtering-data-frontend.md

```diff
@@ -264,33 +312,43 @@ that correspond to the requested page and call
 and return the corresponding `Movie` objects.
 
 ```tsx
-static async fetchPage(connection: web3.Connection, page: number, perPage: number): Promise<Movie[]> {
-  if (this.accounts.length === 0) {
-    await this.prefetchAccounts(connection)
+static async fetchPage(
+  connection: Connection,
+  page: number,
+  perPage: number,
+  reload = false
+): Promise<Movie[]> {
```

Comment:
```suggestion
): Promise<Array<Movie>> {
```

Reply:
Generic syntax (like `Promise<Array<Movie>>`) is clearer (parent child relationship) than `Promise<Movie[]>` - do this everywhere.

Reply:
fixed ✅